README.md

```diff
@@ -106,6 +106,8 @@ The most basic tag type is a simple variable. A `{{name}}` tag renders the value
 
 All variables are HTML-escaped by default. If you want to render unescaped HTML, use the triple mustache: `{{{name}}}`. You can also use `&` to unescape a variable.
 
+To print `{{name}}` as a string literal, the default delimiters must be changed. See the ["Set Delimiter'](https://github.com/janl/mustache.js#set-delimiter) section for more information about custom delimiters.
+
```

Comment:
Yeah, `"{{ name }}"` and `"Bob"` are both string literals.. "Literally in your template" doesn't explain it clearly either, imo. What about mentioning "render"/"output"?

> If you want to render mustache tags – like `{{ name }}` – literally in your output...